h/templates/deform/checkbox.jinja2

```diff
@@ -1,9 +1,19 @@
-<div class="form-checkbox">
+<div class="form-checkbox {% if field.widget.css_class %}{{ field.widget.css_class }}{% endif %}">
   <input type="checkbox"
+         class="form-checkbox__input"
          name="{{ field.name }}"
          id="{{ field.oid }}"
          value="{{ field.widget.true_val }}"
          {%- if cstruct == field.widget.true_val %}
          checked=""
-         {% endif -%}>
+         {% endif -%}
+         {% if field.required %}
+         required
```

Comment:
I discovered a caveat with this that I overlooked before @sheetaluk and @lyzadanger. It makes _every_ checkbox mandatory unless a `missing=False` is specified in the schema. This is because for checkboxes, there is no "empty" state which is different from an "unchecked" state.

What I've done for now is simply to add `missing=False` to the one place where we were using a checkbox that should be uncheckable (the "Trusted" option in the OAuth client creation form).